ext/authx/authx.php

```diff
@@ -230,14 +230,14 @@ function authx_civicrm_permission(&$permissions) {
  *
  * @link https://docs.civicrm.org/dev/en/latest/hooks/hook_civicrm_navigationMenu
  */
-//function authx_civicrm_navigationMenu(&$menu) {
-//  _authx_civix_insert_navigation_menu($menu, 'Mailings', array(
-//    'label' => E::ts('New subliminal message'),
-//    'name' => 'mailing_subliminal_message',
-//    'url' => 'civicrm/mailing/subliminal',
-//    'permission' => 'access CiviMail',
-//    'operator' => 'OR',
-//    'separator' => 0,
-//  ));
-//  _authx_civix_navigationMenu($menu);
-//}
+function authx_civicrm_navigationMenu(&$menu) {
+  _authx_civix_insert_navigation_menu($menu, 'Administer/System Settings', [
+    'label' => E::ts('Configure Authentication'),
```

Comment:
Good idea. Did an `r-run` and worked as expected.

Suggested change in label:

```suggestion
    'label' => E::ts('Authentication'),
```

Note that 9/10 menu-items here are nouns (subjects) rather than verbs (predicates).

<img width="289" alt="Screen Shot 2022-05-30 at 5 17 21 PM" src="https://user-images.githubusercontent.com/1336047/171070393-e32d65c7-3644-4f66-a37b-b94e5caf534a.png">